helix-term/src/commands.rs

```diff
@@ -2637,6 +2637,34 @@ pub mod cmd {
         let (view, doc) = current!(cx.editor);
 
         view.ensure_cursor_in_view(doc, line);
+        Ok(())
+    }
+
+    fn setting(
+        cx: &mut compositor::Context,
+        args: &[Cow<str>],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        let runtime_config = &mut cx.editor.config;
+
+        if args.len() != 3 || args.get(1).map(|s| s.as_ref()) != Some("=") {
+            anyhow::bail!("Bad arguments. Usage: `:set key = field`");
+        }
+
+        match args[0].to_lowercase().as_ref() {
+            "scrolloff" => runtime_config.scrolloff = args[2].parse()?,
```

Comment:
Let's `let arg = &args[2]` before the match statement.

Reply:
:+1: 
https://github.com/helix-editor/helix/pull/798/commits/35a54592f7b96a3e7e8aae21aa6aed69e734e97f